modules/swagger-codegen/src/main/resources/objc/model-body.mustache

```diff
@@ -13,7 +13,11 @@
   }
   return self;
 }
-
+{{#vars}}{{^vendorExtensions.x-unique-id-key}}{{/vendorExtensions.x-unique-id-key}}{{#vendorExtensions.x-unique-id-key}}
+-(NSString*)indexPropertyName {
+    return @"{{name}}";
+}
+{{/vendorExtensions.x-unique-id-key}}{{/vars}}
```

Comment:
Hi,
So I was actually thinking that it would be a good idea to add a unique id for all clients. It’s just an idea but it’s very common for any object to have this kind of property..
But of course I can change it and add the x-objc prefix.


> Wiadomość napisana przez wing328 <notifications@github.com> w dniu 06.06.2016, o godz. 12:31:
> 
> In modules/swagger-codegen/src/main/resources/objc/model-body.mustache <https://github.com/swagger-api/swagger-codegen/pull/3056#discussion_r65868787>:
> 
> > @@ -13,7 +13,11 @@
> >    }
> >    return self;
> >  }
> > -
> > +{{#vars}}{{^vendorExtensions.x-unique-id-key}}{{/vendorExtensions.x-unique-id-key}}{{#vendorExtensions.x-unique-id-key}}
> > +-(NSString*)indexPropertyName {
> > +    return @"{{name}}";
> > +}
> > +{{/vendorExtensions.x-unique-id-key}}{{/vars}}
> If this vendor extension is objc-specified, shall we name it as x-objc-unique-id-key so that it's immediately clear to the others this vendor extension is used by objc client generator?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/swagger-api/swagger-codegen/pull/3056/files/24ecf9d74feb186763f51e55b2afcb9d80a14e2a#r65868787>, or mute the thread <https://github.com/notifications/unsubscribe/ABOar4_2D6Ns3HD3EB-gihNMQYI_S2_Wks5qI_b8gaJpZM4Iuv9t>.
>